Frontend/ui-customer/src/Strings/HI.js

```diff
@@ -429,7 +429,7 @@ const hindiStrings = {
   "NO_DONT": "नहीं",
   "YES_CANCEL_SEARCH": "हाँ, खोज रद्द करें",
   "TRY_LOOKING_FOR_RIDES_AGAIN": "ऐसा लगता है कि यह बहुत ही व्यस्त दिन है। आप फिर से राइड खोजने की कोशिश कर सकते हैं",
-  "NO_TIP": "कोई टिप नहीं",
+  "NO_TIP": "नो टिप",
```

Comment:
previous one was correct

Reply:
Discussed this with Sreedeep.As the UI was breaking because of the text so we changed it.